Workbooks/Windows Virtual Desktop/Connection Performance/Connection Performance.workbook

```diff
@@ -921,7 +921,7 @@
                   "type": 3,
                   "content": {
                     "version": "KqlItem/1.0",
-                    "query": "//aligning to this: https://docs.microsoft.com/en-us/windows-server/remote/remote-desktop-services/rds-rdsh-performance-counters\r\nlet WVDHosts = dynamic([{WVDHosts}]);\r\nPerf\r\n| extend HostPool = '{HostPool:label}'\r\n| extend HostPool = strcat(HostPool, \"/\")\r\n| extend ComputerName = strcat(HostPool, Computer)\r\n| where ComputerName in~ (WVDHosts)\r\n//| where ObjectName == \"RemoteFX Graphics\"\r\n// CounterName in (\"Frames Skipped/Second - Insufficient Server Resources\" \"Frames Skipped/Second - Insufficient Client Resources\")\r\n| where ObjectName == \"RemoteFX Network\" and CounterName in (\"Current TCP RTT\")//, \"Current UDP RTT\")\r\n| where CounterValue > 0\r\n| project TimeGenerated, Computer, _ResourceId, ObjectName, CounterName, InstanceName, CounterValue=toint(bin(CounterValue, 10))\r\n| summarize Samples=count(), (Median, p95)=percentiles(CounterValue, 50, 95), StdDev=sqrt(variance(CounterValue)), (Peak, PeakTime)=arg_max(CounterValue, TimeGenerated), Average=avg(CounterValue) by Host=_ResourceId, Computer=trim_end(@\"\\..*\", Computer)\r\n| project Host, Computer, Samples, Median, Average, p95, StdDev, Peak, PeakTime",
+                    "query": "//aligning to this: https://docs.microsoft.com/en-us/windows-server/remote/remote-desktop-services/rds-rdsh-performance-counters\r\nWVDConnectionNetworkData\r\n| join kind=inner (WVDConnections\r\n    | where _ResourceId =~ \"{HostPool}\"\r\n    | project UserName, _ResourceId, SessionHostName, CorrelationId\r\n)  on CorrelationId\r\n| project-away  CorrelationId1\r\n| summarize percentiles(EstRoundTripTimeInMs, 50, 95), users=dcount(UserName), arg_max(EstRoundTripTimeInMs, TimeGenerated) by Host=trim_end(@\"\\..*\", SessionHostName)\r\n| project Host, users, Median = percentile_EstRoundTripTimeInMs_50, p95=percentile_EstRoundTripTimeInMs_95, Peak = EstRoundTripTimeInMs, PeakTime = TimeGenerated",
```

Comment:
`users=dcount(UserName)`
This becomes the column name in the table, so it should be uppercase

Reply:
Updated the column name to start with an uppercase letter and ensured that all other column names in this workbook are consistently formatted with an initial uppercase.